definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -48,5 +48,49 @@
     "Label": "Reason for refusal",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPayment",
+    "Label": "How will payment be made?",
+    "HintText": "How will payment be made?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBANumber",
+    "Label": "Enter your account number",
+    "HintText": "Example: PBA0896124",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBAReference",
+    "Label": "Enter your reference",
+    "HintText": "This will appear on your statement to help identify your payment",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "HWFReference",
```

Comment:
ditto

Reply:
What does Ditto mean?

Reply:
ditto. :D

Reply:
you're too young to know...
https://www.youtube.com/watch?v=LhkwiJowIUM

Reply:
ditto....
Please check with Jeremy.

Reply:
Waiting on Response from jeremy